ksqldb-engine/src/main/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporter.java

```diff
@@ -267,6 +267,13 @@ public static synchronized BigInteger getMaxTaskUsage(final Metrics metricRegist
   }
 
   private String getQueryId(final KafkaMetric metric) {
+    final String taskName = metric.metricName().tags().getOrDefault("task-id", "");
+    final Pattern namedTopologyPattern = Pattern.compile("(.*?)__(\\d*_\\d*)");
```

Comment:
I'm not an expert on regex, just checking for the last two digits should that be with or without the parentheses?